guides/request-validation-python/example-2/example-2.5.x.py

```diff
@@ -1,9 +1,10 @@
 from flask import Flask, request
 from twilio import twiml
-
+from . import validate_twilio_request
```

Comment:
Actually, this folder `guides/` is not tested at all, so right now I'm thinking it's better to revert its changes, and exclude this folder from the linting.
wdyt?